src/shared/meson.build

```diff
@@ -465,7 +465,8 @@ libshared_deps = [threads,
                   libseccomp,
                   libselinux,
                   libzstd,
-                  libxz]
+                  libxz,
+                  libxenctrl]
```

Comment:
is this still needed, given it's no longer linking?

Reply:
It'd be needed for getting the `-I` flags to access the headers.